### ISSUE TITLE
Refactor PHP client to parse data as actual types

### ISSUE THREAD

**Issue by sbrattla** [stoicflame/enunciate#930]
The PHP XML client currently parses all data as `string`. I'd really like it to at least parse boolean values as `boolean`, but I'm uncertain about the consequences.

If anyone out there use the PHP XML client, they'd might now be doing `isAvailable() === 'true'`, and if I commit a patch for the parser this would turn into a false since `isAvailable()` would not equal a `boolean ` true with `===`.

On the other hand, it's much easier to work with booleans as `boolean`. That would allow me to do `isAvailable()` without the explicit ` === 'true'`.

What's your thoughts on this @stoicflame?

**Comment by stoicflame** [comment]
Yes, variables are passed into the freemarker template via the "model" (it's just a map).

See `PHPXMLClientModule` lines 161-172.
